docker.go

```diff
@@ -133,79 +135,95 @@ func NewContainerHostConfig(opts ...hostOptions) *container.HostConfig {
 
 }
 
+func seccompProfile(userns bool) string {
+	if userns {
+		return seccompWeakProfile
```

Comment:
The seccomp weak profile should be orthogonal to docker using a new userns for the containers. We had 3 basic profiles:

- default docker (with default seccomp, without docker creating a new userns)
- weak docker (with weak seccomp (allowing the container to create new userns), without docker creating a new userns)
- docker + userns (with default seccomp, with docker creating a new userns)

Reply:
That means I can get rid of https://github.com/kinvolk/contained.af/pull/8/commits/58d7ddd08bb022d4f6bce5ad5681989b48b96799 ?

Reply:
I think yes